src/guides/v2.4/install-gde/prereq/php-settings.md

```diff
@@ -41,18 +41,25 @@ Magento requires a set of extensions to be installed:
 <!--{% assign packages = site.data.codebase.v2_4.open-source.composer_lock.packages %}-->
 {% include install/php-extensions-template.md %}
 
-In the command line, type:
+{:.bs-callout-warning}
+If you install Magento by cloning the [magento/magento2](https://github.com/magento/magento2) GitHub repository, you must have the [ext-sockets](https://github.com/php-amqplib/php-amqplib/blob/master/CHANGELOG.md#281---2018-11-13) extension installed on your system.
```

Comment:
```suggestion
If you install Magento 2.3 by cloning the [magento/magento2](https://github.com/magento/magento2) GitHub repository, you must have the [ext-sockets](https://github.com/php-amqplib/php-amqplib/blob/master/CHANGELOG.md#281---2018-11-13) extension installed on your system.
```

Reply:
@meker12: honestly I would remove this paragraph. After some quick research, it's only applicable to Magento 2.3.4 and 2.3.5 (I don't know what 2.3.6 will do in this regards since I don't have access to it yet).
It will be less confusing if we remove this paragraph I think 🙂